### DISCUSSION TITLE
Error: No such file or directory

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/goreleaser#2338] [Q&A] [answered]
I'm using a working setup for a new repo, but I'm getting "`Error: No such file or directory`" for both "Run GoReleaser" and "Post Checkout":

```
0s
Run goreleaser/goreleaser-action@v2
  with:
    version: latest
    args: release --rm-dist
    distribution: goreleaser
    workdir: .
    install-only: false
  env:
    GOROOT: /opt/hostedtoolcache/go/1.15.13/x64
    GITHUB_TOKEN: ***
Error: No such file or directory
0s
Post job cleanup.
Error: No such file or directory
```

![image](https://user-images.githubusercontent.com/422244/124498234-02f88980-dd8a-11eb-9f94-3c2eb941cf4b.png)
https://github.com/go-cc/cc2py/runs/2991848796?check_suite_focus=true

What might be the problem?

Update after 24+hrs: Looks like a bug to me now...

**Comment by caarlos0** [comment]
@crazy-max seems to be on the action... any idea what might have happened?

**Comment by caarlos0** [comment]
Probably unrelated: I think you don't need to set `GOROOT` as the setup go action should do it for you 🤔

**Comment by caarlos0** [comment, marked as answer]
is this still happening?

**Reply by suntong** [reply]
yeah, fixed now.
